src/connector/src/common.rs

```diff
@@ -572,7 +574,8 @@ impl NatsCommon {
     > {
         let context = self.build_context().await?;
         let stream = self.build_or_get_stream(context.clone()).await?;
-        let name = format!("risingwave-consumer-{}-{}", self.subject, split_id);
+        let subject_name = self.subject.replace(",", "-");
+        let name = format!("risingwave-consumer-{}-{}", subject_name, split_id);
```

Comment:
if there are multiple subjects, we may read them in parallel. How about dispatching multiple splits by enumerator?